doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: Uno.GetStarted.Licensing
+---
+
+# Sign in with Uno Platform
+
+Sign in with your Uno Platform account directly in your favorite IDE—Visual Studio, VS Code, or Rider—to unlock powerful tools like Hot Reload, helping you speed up development. With a single registration, you also get early access to new features and the opportunity to connect with the Uno Platform community to share feedback and network.
+
+## Create your account
+
+1. Go to our website, [platform.uno](https://platform.uno/), and click on the **Sign in** button in the top right corner, or go directly to [platform.uno/my-account](https://platform.uno/my-account).
+2. Enter your email address and click on **Register**.
+3. On the registration page, fill in your information. Once done, click on **Sign up**.
+4. You will receive a confirmation email from **no-reply@platform.uno**. Follow the instructions in the email to activate your account.
+5. You should then see the sign-in page. Enter your email and password and click on **Sign in** to access your account details, where you can update information or add more details.
+
+## Sign in to your IDE of choice
+
+After creating your Uno Platform account, follow the steps below to sign in to your preferred IDE:
+
+**I am developing on...**
+
+### [**Visual Studio 2022**](#tab/vs2022)
+
+If you’ve already set up **Visual Studio 2022** by following [Get Started on Visual Studio 2022](xref:Uno.GettingStarted.CreateAnApp.VS2022), sign in as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening and loading completely your project, a popup should appear. Click **Sign in / Register**.
+   ![Visual Studio 2022 Popup](Assets/uno-settings-popup.png)
+   >![NOTE]
+   > If the popup doesn’t appear, access Uno Platform settings by clicking on **Extensions** > **Uno Platform** > **Settings...**.
```

Comment:
@jeromelaban, @kazo0 what should be the proper term to use here please, as I don't think it is named a "popup" here. 
Naggers? Banners? to be consistent, let us know the proper terms to use for each IDE here please

Reply:
In the current docs (which will need to be updated btw) we currently refer to the old licensing nag as a "notification window" for both the [VS Code](https://platform.uno/docs/articles/create-an-app-vscode.html?tabs=Wasm) and [Rider](https://platform.uno/docs/articles/create-an-app-rider.html?tabs=rider2024-2%2Cdesktop) guides.

> A notification window will appear...

Reply:
I feel like "notification" is good enough